lib/Serverless.js

```diff
@@ -68,13 +68,13 @@ class Serverless {
   run() {
     if (!this.cli.displayHelp(this.processedInput) && this.processedInput.commands.length) {
       // trigger the plugin lifecycle when there's something which should be processed
-      return this.pluginManager.run(this.processedInput.commands);
+      this.pluginManager.run(this.processedInput.commands);
```

Comment:
If it's possible, yeah lets move it before that. I'd really hate if the tracking stuff broke some of our working code, since it's not super essential to the framework itself.